database/entity/0075-add_updated_by_contribution/Contribution.ts

```diff
@@ -0,0 +1,104 @@
+import { Decimal } from 'decimal.js-light'
+import {
+  BaseEntity,
+  Column,
+  Entity,
+  PrimaryGeneratedColumn,
+  DeleteDateColumn,
+  JoinColumn,
+  ManyToOne,
+  OneToMany,
+  OneToOne,
+} from 'typeorm'
+import { DecimalTransformer } from '../../src/typeorm/DecimalTransformer'
+import { User } from '../User'
+import { ContributionMessage } from '../ContributionMessage'
+import { Transaction } from '../Transaction'
+
+@Entity('contributions')
+export class Contribution extends BaseEntity {
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
+  id: number
+
+  @Column({ unsigned: true, nullable: false, name: 'user_id' })
+  userId: number
+
+  @ManyToOne(() => User, (user) => user.contributions)
+  @JoinColumn({ name: 'user_id' })
+  user: User
+
+  @Column({ type: 'datetime', default: () => 'CURRENT_TIMESTAMP', name: 'created_at' })
+  createdAt: Date
```

Comment:
I have simply copied the previous entity and added only the field: updatedBy
Do you know if the current code use ms precision for one or all contribution date fields?
To decide if we could use @CreateDateColumn and/or @UpdateDateColumn here I must first check the current code how the fields are used and which problems it could make changing that.

Reply:
Simply a reminder about the ms-dimension.
Perhaps we can fix it as a reminder or technical debt-issue for later refactorings.